dev-docs/analytics/intentiq.md

```diff
@@ -18,3 +18,11 @@ No registration for this module is required.
 <b>IMPORTANT</b>: requires Intent IQ Universal ID module be installed and configured. [(How-To)](https://docs.prebid.org/dev-docs/modules/userid-submodules/intentiq.html)
 
 No additional configuration for this module is required. We will use the configuration provided for Intent IQ Universal IQ module.
+
+#### Example Configuration
```

Comment:
You should also add this as a proper analytics module like https://github.com/prebid/prebid.github.io/blob/master/dev-docs/analytics/agma.md 

and link the module here.

Reply:
Hi, I'm one of the contributors of this branch. Could you please explain what exactly needs to be added here? @muuki88

Reply:
I messed up, sorry :see_no_evil:  I mixed up PR reviews